src/App.js

```diff
@@ -3,8 +3,9 @@ import { BrowserRouter as Router, Route } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
please delete `bootstrap`

Reply:
Sorry, I will remove this later a little bit. At the moment I need it for some classes